generic3g/MAPL_Generic.F90

```diff
@@ -402,6 +407,62 @@ subroutine gridcomp_add_varspec_basic(gridcomp, variable_spec, rc)
       _RETURN(_SUCCESS)
    end subroutine gridcomp_add_varspec_basic
 
+   subroutine gridcomp_add_fieldspec( &
```

Comment:
Just a reminder that we may decide to generalize this to support e.g. TangentVector.  Basically the intent is to provide a simplified interface for the ACG.

And ... we have to keep in mind that there will always be some exceptional specs that require explicit user code which may place additional pressure on the args and/or name here.